Source/WebCore/page/Settings.yaml

```diff
@@ -253,12 +232,6 @@ ImagesEnabled:
     WebCore:
       default: true
 
-IsAccessibilityIsolatedTreeEnabled:
-  type: bool
-  defaultValue:
-    WebCore:
-      default: false
-
```

Comment:
The declaration in UWP.yaml has `webcoreBinding: DeprecatedGlobalSettings`. Does it need to be moved to a per-instance setting there?

Reply:
Good point. I'll double-check that.

Reply:
Lots of code seems to expect it to live in `DeprecatedGlobalSettings`, and don't necessarily have a page context to grab the setting from. I think this is correct as-is.